CMakeLists.txt

```diff
@@ -1,14 +1,11 @@
 cmake_minimum_required(VERSION 3.16)
 project(jluna)
 
-set(CMAKE_CXX_STANDARD 20)
-set(CMAKE_BUILD_TYPE Debug)
-
 # compiler support
 if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION STRGREATER_EQUAL "12.0.0")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++2b -lstdc++")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-volatile")
```

Comment:
While we should definitely surpress the warning, not adding `-lstdc++` here, causes linking to fail by throwing "undefined reference to std::*" on my machine. I don't have much experience with clang, but just removing them doesn't work for me:

```
[100%] Linking CXX executable JLUNA_TEST
/usr/bin/ld: /usr/bin/ld: DWARF error: could not find variable specification at offset 25e81
CMakeFiles/JLUNA_TEST.dir/.test/main.cpp.o: in function `main':
/home/clem/Workspace/jluna/.test/main.cpp:25: undefined reference to `std::allocator<char>::allocator()'
/usr/bin/ld: /home/clem/Workspace/jluna/.test/main.cpp:25: undefined reference to `std::allocator<char>::~allocator()'
/usr/bin/ld: /home/clem/Workspace/jluna/.test/main.cpp:33: undefined reference to `std::allocator<char>::allocator()'
/usr/bin/ld: /home/clem/Workspace/jluna/.test/main.cpp:33: undefined reference to `std::allocator<char>::~allocator()'
(...)
```